### ISSUE TITLE
Project detection failed

### ISSUE THREAD

**Issue by howie6879** [orchest/orchest#116]
Hi there:

Sometimes I delete a project by using `filemanager` and then import it again.

This causes the orchest to fail to load the project, The operation process is as follows:

```shell
cd orchest/userdir
git clone https://github.com/orchest/quickstart
```

 orchest will load this project automatically, The next web operation is as follows:

- Click FileManage->projects
- Delete quickstart dir.

Finally  import `quickstart` project by using Git again. 

At this point, orchest can't load the  `quickstart` project.

You can check the screenshot that I provided for details:

![bd](https://gitee.com/howie6879/oss/raw/master/uPic/bd.gif)

**Comment by howie6879** [comment]
I've found the cause, I'll fix this later

**Comment by ricklamers** [comment]
Thank you @howie6879 for reporting this. Will gladly accept a PR. Either way we’ll make sure this gets fixed asap.

Great find!

**Comment by howie6879** [comment]
> Thank you @howie6879 for reporting this. Will gladly accept a PR. Either way we’ll make sure this gets fixed asap.
> 
> Great find!

Done!

**Comment by ricklamers** [comment]
This is fixed as per #117.
